coremltools/converters/mil/mil/ops/defs/pool.py

```diff
@@ -77,8 +77,8 @@ def type_inference(self):
 @register_op(doc_str="")
 class avg_pool(Pooling):
     """
-    Perform average pooling. Currently defined only for spatial 1-D and
-    2-D cases. (Can be extended to the 3-D case easily.)
+    Perform average pooling. Supports spatial 1-D, 2-D, and 3-D
+    cases.
     
```

Comment:
For this op and other pool ops, update the equation for `D`
`D_in` are spatial dimensions, `1 <= len(D_in) <= 2.` --->   `1 <= len(D_in) <= 3`.

Reply:
OK. See also `ceil_mode` in `avg_pool`, I removed the following bullet point:
`
        * Only supported for a 1D or 2D pool.
`